packages/builders/__tests__/components/actionRow.test.ts

```diff
@@ -5,7 +5,7 @@ describe('Action Row Components', () => {
 	describe('Assertion Tests', () => {
 		test('GIVEN valid components THEN do not throw', () => {
 			expect(() => new ActionRow().addComponents(new ButtonComponent())).not.toThrowError();
-			expect(() => new ActionRow().setComponents([new ButtonComponent()])).not.toThrowError();
+			expect(() => new ActionRow().setComponents(new ButtonComponent())).not.toThrowError();
```

Comment:
this line can be removed now as it's same as line 7.

Reply:
one's addComponent and one's setComponent, the latter according to the refactor should now only take rest parameters so this is fine

Reply:
yeah got it.